diff --git temper.c temper.c
index 5670bb5..5a3040a 100644
--- temper.c
+++ temper.c
@@ -15,6 +15,7 @@ static float offset = -0.85;
 int main(){
        int passes = 0;
        float tempc = 0.0000;
+
        do {
                usb_dev_handle* lvr_winusb = pcsensor_open();

@@ -38,15 +39,7 @@ int main(){
                /* Apply calibrations */
                tempc = (tempc * scale) + offset;

-               struct tm *utc;
-               time_t t;
-               t = time(NULL);
-               utc = gmtime(&t);
-
-               char dt[80];
-               strftime(dt, 80, "%d-%b-%Y %H:%M", utc);
-
-               printf("%s,%f\n", dt, tempc);
+               printf("%f\n", tempc);
                fflush(stdout);

                return 0;

